browser/speedreader/speedreader_tab_helper.cc

```diff
@@ -38,9 +38,10 @@ void SpeedreaderTabHelper::UpdateActiveState(
 
   // Work only with casual main frame navigations.
   if (handle->GetURL().SchemeIsHTTPOrHTTPS()) {
-    auto* whitelist = g_brave_browser_process->speedreader_whitelist();
+    auto* rewriter_service =
+        g_brave_browser_process->speedreader_rewriter_service();
     if (speedreader::IsWhitelistedForTest(handle->GetURL()) ||
-        whitelist->IsWhitelisted(handle->GetURL())) {
+        rewriter_service->IsWhitelisted(handle->GetURL())) {
```

Comment:
Now it's not super clear why `IsWhitelisted` belongs to "rewriter" service. Maybe we could rename to something generic, like `SpeedreaderService`

Reply:
I'd agree, but we already have `SpeedreaderService` that does a very different function of controlling/tracking when the feature is enabled/disabled. Here it `isWhitelisted` for rewriting, so made sense to be more specific